README.md

```diff
@@ -198,13 +202,13 @@ malicious code.
 
 ```shell
 cd ../functionary_carl
-echo "something evil" >> demo-project/foo.py
+echo something evil >> demo-project/foo.py
 ```
 Carl thought that this is the sane code he got from Bob and
```

Comment:
Probably shouldn't be in this PR but maybe okay to sneak in.

sane -> same

Reply:
I actually did mean to write *"sane"*, but since you're the second person (https://github.com/in-toto/demo/pull/3) to stumble across this, I'll change it.

Does *"genuine"*  sound right?

Reply:
Yes, I think genuine could work better than sane.